docs/installation/minikube-amd64.md

```diff
@@ -0,0 +1,191 @@
+
+sudo groupadd docker
+
+sudo usermod -aG docker $USER
+
+# Re-Login and Restart the Docker Server
+systemctl restart docker
+```
+
+#### Step 2：
+- [Install Minikube](https://minikube.sigs.k8s.io/docs/start/) (e.g. Linux amd64)
+    - Requirements：version <= 1.20.x
+```shell
+VERSION=v1.20.0
+curl -LO https://github.com/kubernetes/minikube/releases/download/${VERSION}/minikube-linux-amd64
+sudo install minikube-linux-amd64 /usr/local/bin/minikube
+minikube start --driver=docker
+```
+
+
+
+- Alias `minikube kubectl`
+```shell
+alias kubectl="minikube kubectl --"
+```
+#### Step 3:
+
+- Deploy ingress controller
+```shell
+minikube addons enable ingress
+```
+
+## 3 Deploy
+
+### 3.1 Modify Helm Configuration
+
+#### Step 1：
+- Install helm:
+
+Please refer to [helm installation](https://helm.sh/docs/intro/install/)
+
+- Download Helm Repo:
+
+``` shell
+git clone git@github.com:tkestack/kstone.git
+cd ./charts
+```
+
+#### Step 2：
+
+- Get Admin TOKEN from Minikube cluster
+
+```shell
+kubectl get secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='default')].data.token}" -n kube-system|base64 --decode
+```
+
+- Fill in the TOKEN of the cluster to deploy.
+
+``` yaml
+// charts/charts/dashboard-api/values.yaml
+
+kube:
+  token: $token
+  target: kubernetes.default.svc.cluster.local:443
+```
+
+- Requirements：
+    - $token is the access credential TOKEN of the cluster to be deployed.
+    - $token needs to have access to all resources in the cluster.
+
+#### Step 3: Using the existing Prometheus Operator (optional)
+
+- Set `replica=0` in the file `charts/charts/kube-prometheus-stack/values.yaml`.
+- Modify the file: `charts/charts/grafana/templates/configmap.yaml`, replace `http://{{ .Release.Name }}-prometheus-prometheus.{{ .Release.Namespace }}.svc.cluster.local:9090` to the query URL from the existing Prometheus Operator.
+
+### 3.2 Install
+
+- Install Kstone
+
+``` shell
+cd charts
+
+kubectl create ns kstone
+
+helm install kstone . -n kstone
+```
+- Create Ingress Rule
+  
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: kstone-ingress
+  namespace: kstone
+spec:
+  rules:
+    - http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: kstone-dashboard
+                port:
+                  number: 80
+          - path: /apis
+            pathType: Prefix
+            backend:
+              service:
+                name: kstone-dashboard-api
+                port:
+                  number: 80
+          - path: /grafana
+            pathType: Prefix
+            backend:
+              service:
+                name: kstone-grafana
+                port:
+                  number: 80
+EOF
+```
+
+- Expose Ingress Port
+```shell
+kubectl port-forward deployment/ingress-nginx-controller 8080:80 --namespace kube-system --address 0.0.0.0
```

Comment:
i got Error "from server (NotFound): deployments.apps "ingress-nginx-controller" not found".
namespace should be ingress-nginx. or is it related to your minikube version?